dp/generate_climos.py

```diff
@@ -39,17 +39,96 @@
 cdo = Cdo()
 
 
+def input_check(filepath, climo):
+    '''
+    This function runs general checks on the given input arguments filepath and climo.
+    First, it checks if the given input file path is a vaild NetCDF file. It returns
+    NetCDF dataset of the input file. Ohterwise, it halts the program. Next, it checks
+    if the input NetCDF dataset has a set of standard climo periods by using climo_periods
+    property. It returns the periods if found, outherwise it halts the program.
```

Comment:
It doesn't actually halt the program, it just returns (and see comments below about _what_ to return). The behaviour is correct, but the description needs a little tweaking.

Also, we should document what `climo` is: which is, I believe a list of strings naming climo periods.